pom.xml

```diff
@@ -100,9 +99,30 @@
           <compatibleSinceVersion>1.3.0</compatibleSinceVersion>
         </configuration>
       </plugin>
+
+      <plugin>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>display-info</id>
+            <configuration>
+              <rules>
+                <requireUpperBoundDeps>
+                  <excludes combine.children="append">
+                    <!--for org.jenkins-ci.plugins:matrix-auth:1.16-->
+                    <exclude>org.jenkins-ci.plugins.icon-shim:icon-set</exclude>
```

Comment:
Fine for now, it is a wider issue in Jenkins. The discussion is here: https://github.com/jenkinsci/plugin-pom/pull/72